### ISSUE TITLE
inplace operation

### ISSUE THREAD

**Issue by Sam-Yuan** [leftthomas/SRGAN#93]
one of the variables needed for gradient computation has been modified by an inplace operation: [torch.cuda.FloatTensor [1, 1024, 1, 1]] is at version 2; expected version 1 instead. Hint: the backtrace further above shows the operation that failed to compute its gradient. The variable in question was changed in there or anywhere later. Good luck!

How can i deal with? please

**Comment by YoadTew** [comment]
I just got this error too when I tried to train.
Pytorch version is 1.7.1, GPU is rtx 3070, OS is unbuntu 20.04

**Comment by Sam-Yuan** [comment]
It maybe because of the version of pytorch(my version is 1.5 gtx1660ti windows).
I debuged step by step, but i can' t find any problems﻿😭 if you deal with it successfully, please tell me how should I do. Thank you.



---Original---

**Comment by leftthomas** [comment]
@Sam-Yuan 
Duplicate of #89
